static/openapi.json

```diff
@@ -534,4 +549,4 @@
       }
     }
   }
```

Comment:
https://github.com/internetarchive/openlibrary/pull/8085/files

The ⛔ [below line 552](https://github.com/internetarchive/openlibrary/pull/8085/files) is because the GitHub Editor wants all text files to end with one and only one `\n`.